src/main/java/org/isf/disease/dto/DiseaseDTO.java

```diff
@@ -32,32 +32,32 @@
 public class DiseaseDTO {
 
 	@NotNull
-	@Schema(description = "Disease code", example = "99")
+	@Schema(description = "Disease code", example = "99", maxLength = 10)
 	private String code;
 
 	@NotNull
-	@Schema(description = "Disease description")
+	@Schema(description = "Disease description", maxLength = 50)
```

Comment:
the next line looks like the indentation is wrong.

Reply:
please for this i don't see the wrong indentation

Reply:
It is not line 39 but line 40 that looks wrong; I could not put a comment on that line and that is the reason I wrote:  "the next line looks....."

Reply:
OKay I think it is okay now